cloud2cloud-gateway/config.yaml

```diff
@@ -0,0 +1,83 @@
+log:
+  debug: false
+apis:
+  http:
+    address: "0.0.0.0:9100"
+    ownerClaim: "sub"
```

Comment:
I expected it under `clients.authorizationServer.ownerClaim`

Reply:
It's not needed at all after I've removed the `CtxWithOwner` calls.